internals/ci/src/cmds/version/index.ts

```diff
@@ -7,9 +8,51 @@ import { expectSuccess } from "@oko-wallet-ci/expect";
 import { sleep } from "@oko-wallet-ci/time";
 import { doBuildSDK } from "../build_sdk";
 
+function checkWorkspaceVersions() {
+  console.log('Checking for "workspace:" versions in publishable packages...');
+
+  const result = spawnSync(
+    "grep",
```

Comment:
@Ryz0nd This is a brilliant attempt to pre-determine whether there is a "workspace: " dependencies. However, I am generally against the idea of using an external binary such as "grep" from our system. This script can run in an environment where there is no grep.

Could we retrieve and even "read" all the package.json s in the repository then figure out if there's workspace:* ?